SeaORM/versioned_docs/version-0.10.x/03-migration/02-writing-migration.md

```diff
@@ -205,3 +205,58 @@ In the grand scheme of things, we recommend a schema first approach: you write m
 At times, you might want to use the [`create_*_from_entity`](09-generate-sea-query-statement/01-create-table.md) methods to bootstrap your database with several hand written entity files.
 
 That's perfectly fine if you intend to never change the entity schema. Or, you can keep the original entity and embed a copy in the migration file.
+
+## Combining Multiple Schema Changes in one Migration
+
+You can combine multiple changes within both up and down migration functions. For example here we create both a table and an index in the up:
+
+   ```rust
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+
+        manager
+            .create_table(
+                sea_query::Table::create()
+                    .table(Post::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Post::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key()
+                    )
+                    .col(ColumnDef::new(Post::Title).string().not_null())
+                    .col(ColumnDef::new(Post::Text).string().not_null())
+                    .to_owned()
```

Comment:
perhaps we can also `...` it as it is a duplicate of the above section?

Reply:
Apologies if I pulled it into the wrong branch. Which branch should I pull it into? (my first time)

As a newbie to Rust and SeaOrm (with a decade away from much coding) seeing the whole code, even if it is a repeat is helpful. I sometimes find the little snippets a bit confusing in terms of where they fit into a project.

Reply:
> Apologies if I pulled it into the wrong branch. Which branch should I pull it into? (my first time)
It's fine. I meant https://github.com/SeaQL/seaql.github.io/blob/master/SeaORM/docs/03-migration/02-writing-migration.md

> As a newbie to Rust and SeaOrm (with a decade away from much coding) seeing the whole code, even if it is a repeat is helpful. I sometimes find the little snippets a bit confusing in terms of where they fit into a project.

Right

Reply:
I think I have now created a pull request for that version.